Sources/Tentacle/RepositoryInfo.swift

```diff
@@ -63,6 +60,65 @@ public struct RepositoryInfo: CustomStringConvertible, Identifiable {
     public var description: String {
         return nameWithOwner
     }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.id = try container.decode(ID.self, forKey: .id)
+        self.owner = try container.decode(UserInfo.self, forKey: .owner)
+        self.name = try container.decode(String.self, forKey: .name)
+        self.nameWithOwner = try container.decode(String.self, forKey: .nameWithOwner)
+        self.body = try container.decodeIfPresent(String.self, forKey: .body)
+        self.url = try container.decode(URL.self, forKey: .url)
+        self.homepage = try container.decodeIfPresent(URL.self, forKey: .homepage)
```

Comment:
This doesn't work for empty string. That should be why `try?` was used here.

Reply:
That doesn't totally match the intent, since a string that's not a URL should be `nil`, but a number should throw. But it's probably close enough. 🤷‍♂️

Reply:
Soooo, back to `try?` is what you meant?

Reply:
Yeah, just for `homepage`.

Reply:
Done!